src/status_im/ui/screens/ens/views.cljs

```diff
@@ -357,8 +357,8 @@
       (stateofus/subdomain username)
       [{:style {:color colors/gray}}
        (i18n/label :t/ens-saved)]]]
-    [react/view {:flex-direction :row :margin-left 6 :margin-top 14 :align-items :center}
-     [react/text {:style {:text-align-vertical :center}}
+    [react/view {:flex-direction :row :margin-left 6 :margin-top 14 :align-items :center :justify-content :center}
```

Comment:
Sorry, was unclear - when flex-direction is `row` then `align-items` does the trick.